src/borg/testsuite/helpers.py

```diff
@@ -800,7 +800,9 @@ def test_get_runtime_dir(monkeypatch):
     else:
         monkeypatch.delenv("XDG_RUNTIME_DIR", raising=False)
         monkeypatch.delenv("BORG_RUNTIME_DIR", raising=False)
-        assert get_runtime_dir() == os.path.join("/run/user", str(os.getuid()), "borg")
+        assert get_runtime_dir() in [os.path.join("/run/user", str(os.getuid()), "borg"),
+                                     os.path.join("/var/run/user", str(os.getuid()), "borg"),
+                                     os.path.join("/tmp/runtime-" + str(os.getuid()), "borg")]
```

Comment:
thanks for the update!

please do some small changes:
- `black` formatting check is failing (run black locally, see our developer docs, and commit)
- extract the `os.getuid()` and use an f-string instead of `+`

Reply:
Just force pushed an update that addresses your feedback.